client/components/editor/teaching-elements/Assessment/TextSelection/helpers.js

```diff
@@ -0,0 +1,110 @@
+import Blast from 'blast-vanilla';
+import first from 'lodash/first';
+import generateRange from 'lodash/range';
+import isEmpty from 'lodash/isEmpty';
+import last from 'lodash/last';
+import forEach from 'lodash/forEach';
+
+const blastClass = 'blast';
+const wordClass = 'word';
+const idPrefix = 'text-';
+
+const isText = node => node.nodeType === Node.TEXT_NODE;
+
+export function extractIndex(node) {
+  const [, index] = node.id.split('-');
```

Comment:
```js
const [_, index] = node.id.split('-');
```
triggered lint error or there is another reason you decided omit placeholder param listing? 🤔

Reply:
`[_, index]` syntax is triggering lint errors because `_` is unused.

Reply:
Bad boy ESLint, bad boy 😁